ui/src/app/pages/data/cohort/demographics.tsx

```diff
@@ -269,11 +269,8 @@ export class Demographics extends React.Component<Props, State> {
     this.setState({ ageType }, () => this.calculateAgeFromNodes());
   }
 
-  onSliderInit(slider: React.ReactNode) {
```

Comment:
now there's a second problem: it's also not a ReactNode!  The easiest solution seems to be to keep this untyped

Reply:
Tested this: Cohort -> Demographics -> Age

The age slider and count have the same behavior as before.